go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/sonr-io/sonr
 
-go 1.16
+go 1.17
 
 require (
-	github.com/99designs/keyring v1.1.6
```

Comment:
This dep should not have been removed. I ran `go mod tidy` locally with your current changes and it automatically added it back, can you try this?

Reply:
fixed